vessim/cosim/computing_system.py

```diff
@@ -26,6 +26,15 @@ def init(self, sid, time_resolution, step_size, eid_prefix=None):
         self.step_size = step_size
         return super().init(sid, time_resolution, eid_prefix=eid_prefix)
 
+    def finalize(self) -> None:
+        """Stops http power meters' threads."""
+        super().finalize()
+        for model_instance in self.entities.values():
+            for power_meter in model_instance.power_meters:
+                if isinstance(power_meter, HttpPowerMeter):
+                    power_meter.update_thread.stop()
+                    power_meter.update_thread.stop()
+
```

Comment:
The update threads of the power meters need to be stoppped somewhere when the simulation finishes. Since this is the only mosaik simulator that accesses the power meters, I figured this should be fine for now, however is this how we want to do this?